### DISCUSSION TITLE
How to use --files-from options ?

### DISCUSSION THREAD

**Discussion by Stephenfang51** [RsyncProject/rsync#264] [Q&A] [answered]
I want to move some specific files store in test.txt line by line from host to remote using this function, but How?

` rsync -avzh --files-from="test.txt" /home/user/path/tomove`, Is it correct ?

never suceed..

**Comment by WayneD** [comment, marked as answer]
You must specify a source and destination directory for even a files-from copy.  The paths in the files-from list are all relative to the source directory.  Specify "/" as the source dir if you want to specify full, absolute paths in the file.  Also, don't specify -z on local copies (I also recommend -i). So, you probably want to specify:

`rsync -aivh --files-from=test.txt / /home/user/path/tomove`
or
`rsync -aivh --files-from=test.txt . /home/user/path/tomove`

**Reply by Stephenfang51** [reply]
Thanks for answering the question.
